py4DSTEM/process/utils/ellipticalCoords.py

```diff
@@ -1,100 +1,200 @@
-# Contains methods relating to polar-elliptical calculations.
-#
-# This includes
-#   - measuring / fitting elliptical distortions
-#   - transforming data from cartesian to polar-elliptical coordinates
-#   - correcting Bragg peak positions for elliptical distortions
-#   - converting between ellipse representations
-#
-# We define the transformation from cartesian to polar-elliptical coordinates by:
-#
-#       x = x0 + A*r*cos(theta)*cos(phi) + B*r*sin(theta)*sin(phi)
-#       y = y0 + B*r*sin(theta)*sin(phi) - A*r*cos(theta)*cos(phi)
-#
-# All angular quantities are in radians.
+"""
```

Comment:
I feel like this file should be renamed to elliptical_coords.py just to be consistent, and what better time than now.